gems/smithy/spec/interfaces/client/waiters_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require_relative '../../spec_helper'
+
+describe 'Client: Waiters' do
+  let(:input) { { string_property: 'input_string' } }
+  let(:client) { Wait_Service::Client.new(stub_responses: true) }
+  let(:unexpected_error) { Smithy::Client::Waiters::Errors::UnexpectedError }
+  let(:no_such_waiter_error) { Smithy::Client::Waiters::Errors::NoSuchWaiterError }
+
+  ['generated client gem', 'generated client from source code'].each do |context|
+    context context do
+      include_context context, 'Wait_Service'
+
+      it 'returns when successful' do
+        output = {}
+        expect(client).to receive(:get_operation).and_return(output)
+        expect {
```

Comment:
Do .. end for multiline blocks
Or you can put it on one line and start the next line with .to